torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1423,11 +1419,12 @@ def _setup_streams(self) -> None:
                 m._streams = self._streams
                 m._fsdp_graph_order = self._fsdp_graph_order
                 # Give each non-root FSDP module an alias to the root's
-                # execution order data structure and ignored parameter and
-                # buffer names since only the root's names are fully prefixed
-                # like the state dict keys
+                # execution order data structure and the root's ignored
+                # parameters and all buffer names since only the root's names
```

Comment:
```suggestion
                # parameter names and all buffer names since only the root's names
```
I need to fix this grammatical typo I introduced last night, but I am not going to re-push and re-trigger CI just for this. (Leaving this comment as a reminder to do so.)